doc/whats_new/upcoming_changes/sklearn.linear_model/32114.api.rst

```diff
@@ -0,0 +1,15 @@
+- :class:`linear_model.LogisticRegressionCV` got a new parameter
+  `use_legacy_attributes` to control the types and shapes of the fitted attributes
+  `C_`, `l1_ratio_`, `coefs_paths_`, `scores_` and `n_iter_`.
+  The current default value `True` keeps the legacy behaviour. If `False` then
+    - `C_` is a float
```

Comment:
```suggestion
  The current default value `True` keeps the legacy behaviour. If `False` then:

    - `C_` is a float
```

Reply:
I am not successful in getting the rendering right.